src/os/host-start.r

```diff
@@ -423,6 +421,8 @@ host-start: function [
         ]
     ]
 
+    boot-embedded: get-encap system/options/boot
+
```

Comment:
system/options/boot might not point to the executable path, and [here](http://stackoverflow.com/questions/933850/how-do-i-find-the-location-of-the-executable-in-c) is the discussion about how to do it in C